specification/advisor/resource-manager/Microsoft.Advisor/preview/2022-02-01-preview/examples/Predict.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "subscriptionId": "subscriptionId",
+    "predictionRequest": {
+      "properties": {
+        "predictionType": "PredictiveRightsizing",
+        "extendedProperties": {
+          "region": "CentralUS",
+          "deploymentType": "Linux_IaaS_Software_Store",
+          "sku": "Standard_Dv4",
+          "type": "iaas",
+          "numberOfInstances": 10
+        }
+      }
+    },
+    "api-version": "2022-02-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subscriptionId/providers/Microsoft.Advisor/predict/default",
```

Comment:
>default [](http://example.com/codeflow?start=81&length=7)

What does this id represent ? Are you creating a default resource using the POST operation. please note that for ARM we do not allow resource creation via POST actions ..

Reply:
We are not actually creating any resources. This is used for getting a prediction. Since it requires passing parameters in the body of the request, we are using POST.

Reply:
As discussed can you remove these 3 properties.